rfc000-rfc-process.md

```diff
@@ -124,6 +136,7 @@ RFC: unassigned
 Author: Alan Smithee <asmithee@example.com>
 Status: Draft
 Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
```

Comment:
My issue with that is that RFC-000 pretty much *is* the documentation, and explains further down that RFCs can be "Replaces" but doesn't then include that as a valid field in the metadata. I think something like adding <Replaces> to indicate it's an optional would be enough, but not including it at all seems to me to make things a little more opaque.